pkg/vectorize/eq/eq.go

```diff
@@ -1265,10 +1266,21 @@ func float32EqNullableScalarSels(x float32, ys []float32, nulls *roaring.Bitmap,
 	return rs[:rsi]
 }
 
+const tolerance = .00001
```

Comment:
we have this BVT test case that uses float32 comparison after type cast float32 to float64, so I added this almost equal comparison operation, to survive the BVT test?

No! I think it is reasonable to compare float number in this manner, although this method slows down the execution, we will advocate using decimal type for float number comparison anyway once we have decimal type ready.

Reply:
Or, we can use a much more simplified method for float type equare comparison, math.Abs(x - y) < epsilon,
whereas epsilon is a small tolerance like 0.00001.